documentation/getting_started_guide/scripts/process_data.py

```diff
@@ -0,0 +1,141 @@
+from bioio import BioImage
+import pandas as pd
```

Comment:
This script is identical to the steps described in `GETTING_STARTED.md`.

Reply:
combined with getting_started as jupyter ipynb file, you'll be able to remove this